gerrit-cs-status/src/main/java/com/tw/go/plugin/provider/GerritProvider.java

```diff
@@ -59,8 +56,8 @@ public String pluginId() {
     }
 
     @Override
-    public String pollerPluginId() {
-        return GERRIT_CS_POLLER_PLUGIN_ID;
+    public List<String> pollerPluginIds() {
+        return Collections.singletonList(GERRIT_CS_POLLER_PLUGIN_ID);
```

Comment:
Please use static import where-ever possible

Reply:
Fixed in https://github.com/gocd-contrib/gocd-build-status-notifier/pull/134/commits/40e52f5be9411fc11004218ffff2546e0112761b